taskcluster/docker/base/Dockerfile

```diff
@@ -0,0 +1,64 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+FROM ubuntu:18.04
+
+MAINTAINER Tom Prince "mozilla@hocat.ca"
```

Comment:
This should probably be updated (or removed, I'm not sure how much value it is providing).

Reply:
https://github.com/mozilla-extensions/xpi-manifest/pull/32 . (The tip of this PR doesn't have any `taskcluster/docker`; we use the docker image indexes from `xpi-manifest`. I tried copying the docker directory as part of trying to resolve the `--root` bustage.)